src/creatures/players/player.cpp

```diff
@@ -9572,6 +9572,53 @@ void Player::sendLootContainers() const {
 	}
 }
 
+// OTCR Features
+
+void Player::sendAttachedEffect(const std::shared_ptr<Creature> &creature, uint16_t effectId) const {
+	if (!client || !creature) {
+		return;
+	}
+	if (creature->getPlayer()) {
+		if (client) {
```

Comment:
You don't need to check the "client" as it is already checked at the beginning of the function, it is redundant